src/pages/groupComparison/styles.module.scss.d.ts

```diff
@@ -0,0 +1,18 @@
+declare const styles: {
+  readonly "buttonExcludedFromAnalysis": string;
+  readonly "buttonUnselected": string;
+  readonly "comparisonGroupManager": string;
+  readonly "groupButtons": string;
+  readonly "groupCheckboxes": string;
+  readonly "groupItem": string;
+  readonly "groupLineItemActionButtons": string;
+  readonly "groupRow": string;
+  readonly "headerControls": string;
+  readonly "markedForDeletion": string;
+  readonly "noGroupsMessage": string;
+  readonly "survivalPlotHeader": string;
+  readonly "survivalPlotHeaderContainer": string;
+  readonly "survivalTypeOptions": string;
```

Comment:
is this auto created or do we have to go in and add this every time we add a class?

Reply:
auto created, and works fine in the watch mode.